docs/release-notes.md

```diff
@@ -3,6 +3,32 @@
 - Table of Contents
 {:toc}
 
+## Version 1.3.0
+
+The 1.3.0 release of MCUboot brings in many fixes and updates.  There
+are no breaking changes in functionality.  Many of the changes are
+refactorings that will make the code easier to maintain going forward.
+In addition, support has been added for encrypted images.  See [the
+docs](encrypted_images.md) for more information.
+
+### About this release
+
+- Modernize the Zephyr build scripts.
+- Add a `ptest` utility to help run the simulator in different
+  configurations.
+- Migrate the simulator to Rust 2018 edition.  The sim now requires at
+  least Rust 1.32 to build.
+- Simulator cleanups.  The simulator code is now built the same way
+  for every configuration, and queries the MCUboot code for how it was
+  compiled.
+- Abstract logging in MCUboot.  This was needed to support the new
+  logging system used in Zephyr.
+- Add multiple flash support.  Allows slot1/scratch to be stored in an
+  external flash device.
+- Add support for encrypted images.
+- Add support for flash devices that read as '0' when erased.
+- Add support to Zephyr for the `nrf52840_pca10059`
```

Comment:
We could mention that for this board we support Serial Recovery over USB with CDC ACM.

Reply:
Also can mention something like "imgtool is now also available as a python package on pypi.org"

Reply:
+ Add an option to erase flash pages progressively during recovery upload to avoid possible timeouts (required especially by virtual com USB solution)

Reply:
+ Add key unwrap functionality using tinycrypt (apart form mbedTLS available before)

Reply:
+ imgtool: big-endian support
+ imgtool: save to format of given output filename (intel-hex or raw binary)